packs/classfeatures/air-gate.json

```diff
@@ -255,6 +255,26 @@
                     "junction:air:impulse",
                     "air"
                 ],
+                "selector": "impulse-attack-roll",
+                "text": "PF2E.SpecificRule.Kineticist.Impulse.ImpulseJunction.Air",
+                "title": "PF2E.SpecificRule.Kineticist.Impulse.ImpulseJunction.Title"
+            },
+            {
+                "key": "Note",
+                "predicate": [
+                    {
+                        "not": "self:action:slug:elemental-blast"
+                    },
+                    {
+                        "gte": [
+                            "action:cost",
+                            2
+                        ]
+                    },
+                    "impulse",
```

Comment:
Avoid using top-level roll options

Reply:
That is copied from the existing note but I can change them to `self:action:trait:x` if you want.

Reply:
Yeah, that'd be better